tests/functional/testOnboarding.js

```diff
@@ -277,4 +279,65 @@ describe('Onboarding', function() {
     assert.equal(await vpn.getQueryProperty(queries.screenOnboarding.ONBOARDING_VIEW, 'currentIndex'), 3);
     assert.equal(await vpn.getSetting('onboardingStep'), 3);
   });
+
+  it.only('Verify data collection checkbox and setting are decoupled', async () => {
```

Comment:
I've made this same mistake so many times!

```suggestion
  it('Verify data collection checkbox and setting are decoupled', async () => {
```

Reply:
If `it.only` is being used for debugging reasons, we have the option to use mocha's `--grep` option to run a single test case instead, no code changes required. Just do `npm run functionTest path/to/test.js -- --grep "decoupled"` for example.

Reply:
🫠